hemres/models.py

```diff
@@ -10,20 +10,31 @@
 from django.utils.encoding import python_2_unicode_compatible
 from django.utils.safestring import mark_safe
 from janeus import Janeus
-from mezzanine.core.models import SiteRelated
+import hashlib
+import os
 import re
 import bleach
 import html2text
 
 
+try:
+    from mezzanine.core.models import SiteRelated
+except:
+    from .siterelated import SiteRelated
```

Comment:
Maybe add a comment to why this is here? It may not be clear to a reader somewhere in the future.